lx_annotate/settings_prod/__init__.py

```
@@ -19,7 +19,7 @@
     STATICFILES_DIRS
 )
 
-BASE_DIR = Path(__file__).resolve().parent.parent.parent
+BASE_DIR = Path(__file__).resolve().parent.parent
```

Ruff (0.8.2)

Redefinition of unused `BASE_DIR` from line 5

(F811)